source/Nevermore/CommandParameterValues.cs

```diff
@@ -139,4 +162,139 @@ public void AddRange(CommandParameterValues other)
             }
         }
     }
+
+    public class MyDataReader : DbDataReader
```

Comment:
What's the data reader for?

Reply:
Woops didn't mean to include that, thanks